src/app.js

```diff
@@ -29,34 +35,42 @@ export default (
   ): mixed => {
     if (request.method === 'OPTIONS') {
       response.set({
-        'Access-Control-Allow-Headers':
-          'X-Requested-With, Content-Type, Accept, Authorization',
+        'Access-Control-Allow-Headers': 'X-Requested-With, Content-Type, Accept, Authorization',
         'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, OPTIONS',
         'Access-Control-Allow-Origin': '*',
         'Access-Control-Max-Age': '300',
       });
       return response.sendStatus(204);
     }
-    response.set({ 'Access-Control-Allow-Origin': '*' });
+    response.set({
+      'Access-Control-Allow-Origin': '*',
+    });
     return next();
   };
 
   if (settings.LOG_REQUESTS) {
```

Comment:
Sorry my fault. I though you mean CORS(the replaced lines)
I have not remove any code or function right now, but I agree, that doesn't make sense anymore